### PR TITLE
4 5 post details post card

### PR DESCRIPTION
**Pull request by HarryWickham**
Created Post Card component for desktop and mobile.
Created Post Details Component

### CODE REVIEW

File: src/components/Posts/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          title={title}
+          date={date}
+          category={category}
+          excerpt={excerpt}
+          categorySlug={categorySlug}
+          slug={slug}
+        />
+      ) : null}
+    </>
+  );
```
Comment (4dambarr) [inline src/components/Posts/index.tsx:37]:
```suggestion
  return (
    <>
      {isDesktop ? (
        <PostCardDesktop
          image={image}
          title={title}
          date={date}
          category={category}
          excerpt={excerpt}
          categorySlug={categorySlug}
          slug={slug}
        />
      ) : (
      <PostCardMobile
          image={image}
          title={title}
          slug={slug}
          date={date}
          category={category}
          categorySlug={categorySlug}
        />
      )}
    </>
  );
```

**Review by 4dambarr — CHANGES_REQUESTED** [review]

File: src/components/Posts/PostCardDesktop.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+const PostExcerpt = styled(Text, {
+  display: "flex",
+  fontSize: "$2",
+  paddingBottom: "5px",
+  color: "$text",
+  overflow: "hidden",
+  textOverflow: "ellipsis",
+  whiteSpace: "normal",
+  margin: 0,
```
Comment (4dambarr) [inline src/components/Posts/PostCardDesktop.tsx:45]:
```suggestion
  margin: 0,
  
  "&:hover@: {
  cursor: pointer
  }
```

**Review by 4dambarr — CHANGES_REQUESTED** [review]

**Review by 4dambarr — DISMISSED** [review]

File: src/components/Posts/PostCardDesktop.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  height: "145px",
+});
+
+const PostTitle = styled(H2, {});
+
+const PostExcerpt = styled(Text, {
+  display: "flex",
+  fontSize: "$2",
+  paddingBottom: "5px",
+  color: "$text",
```
Comment (benslv) [inline src/components/Posts/PostCardDesktop.tsx:41]: Think this needs to be `$textLight` instead?

**Review by benslv — CHANGES_REQUESTED** [review]
Just one styling change I noticed, otherwise looks good to me!

**Review by benslv — APPROVED** [review]

**Pull request merged**